controllers/cloud.redhat.com/namespacereservation_controller.go

```diff
@@ -319,3 +329,17 @@ func hardCodedUserList() map[string]string {
 		"system:authenticated": "Group",
 	}
 }
+
+func ParseDurationTime(duration *string) (time.Duration, error) {
```

Comment:
Is there a reason to make this public?

Reply:
There isn't. Didn't think about this, I'll update it.